kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -53,7 +53,13 @@ data:
           # If enabled will also stitch together container logs split by docker/cri due to size limits(16KB per log line) up to 64 KB.
           # enabled = "false"
           # stacktrace_languages = []
-
+       #[log_collection_settings.metadata_collection]
+          # if enabled will collect kubernetes metadata for ContainerLogv2 schema. Default false
+          # enabled = false
```

Comment:
Looks like this is not working on MSI clusters, did you verify that, it works on MSI clusters?